### ISSUE TITLE
Craver Terraformation Module on Behemoth Non-Functional

### ISSUE THREAD

**Issue by Reithan** [captaincobbs/ESG-Mod#1565]
It initially said I needed to research Climate Engineering, but after researching, it's still greyed out - but has an error message.
![image](https://github.com/user-attachments/assets/34eeb03f-9274-4974-9a49-3782392ffa55)
I also don't seem to be able to use it to do normal terraformation, either, as its description suggests I should be able to?

**Comment by Reithan** [comment]
This was the original tooltip:
![image](https://github.com/user-attachments/assets/6893a53e-e9b6-497c-92d4-50f988d57d70)

**Comment by Reithan** [comment]
Terraform only show up on behemoth options if it's in sleep mode, but can't be used in sleep mode.
![image](https://github.com/user-attachments/assets/3d2c5713-aea6-49a7-a73e-cef9866693db)
![image](https://github.com/user-attachments/assets/0189f966-eace-4cfc-af69-deb7b12bd902)

**Comment by wakystuf** [comment]
Which module is this? I thought all behemoth terraforming had to be disabled due to an unfixable bug

**Comment by Reithan** [comment]
This one
![image](https://github.com/user-attachments/assets/18539353-ac01-47e7-b6fc-6762870bad06)

**Comment by Reithan** [comment]
I mean if terraforming is disabled, that's working... ish... because you can only see the terraform button while on "Sleep" mode, and you can't press it while sleeping, so you can never press it.

But the Restore Planet button is still busted. :(

**Comment by wakystuf** [comment]
Yeah none of those effects is even terraforming, I don't believe, though I forget the difference between reformation and reconstruction. When you say Restore Planet, do you mean fixing one that has been core cracked?

**Comment by Reithan** [comment]
I'm not sure which is which, but one is for fixing core cracked planets, the other is for fixing depleted planets. I'm specifically trying to do the fix depletion one. I haven't testing fixing core-cracked, yet.

You can see on the tooltip it says, "Turn a depleted planet into a viable one"
![image](https://github.com/user-attachments/assets/06c10f92-6e60-4584-a286-f6cbd8f8ed07)

**Comment by captaincobbs** [comment]
I've been waiting for a vanilla fix for this issue for years, I've been told we'll probably get one soon
